test/CadlRanchProjects/arrays/item-types/Generated/cadl.json

```diff
@@ -1373,7 +1373,7 @@
         "ElementType": {
          "$id": "148",
          "Name": "duration",
-         "Kind": "DurationISO8601",
+         "Kind": "Duration",
```

Comment:
This would be problematic as we won't know what format to serialize back into in a round trip scenario

Reply:
yeah now that we're adding a new format for duration to tcgc, i see the issue. I think it should be a separate field though: `kind` should still be duration, but there should be a `format` property as well